src/browser/BrowserService.cpp

```diff
@@ -709,7 +709,7 @@ QJsonObject BrowserService::showPasskeysAuthenticationPrompt(const QJsonObject&
         return getPublicKeyCredentialFromEntry(entries.first(), publicKey, origin);
     }
 
-    const auto timeout = publicKey["timeout"].toInt();
+    const auto timeout = browserPasskeys()->getTimeout(userVerification, publicKey["timeout"].toInt());
```

Comment:
This function is rather opaque. Is it just checking for 0 and returning X? Why not just do that?

Reply:
IIRC there are different default timeout values, depenging on the `userVerification`.

Reply:
You are right, looks good